docs/modeling-guide.md

```diff
@@ -278,4 +278,67 @@ class FooContainer {
   baz: BazDefinition
   faz: FazDefinition
 }
+```
+
+### Additional information
+
+If needed, you can specify additional information on each type with the approariate JSDoc tag.
+Following you can find a list of the supported tags:
+
+#### `@since`
+
+Every API already has a `@since` tag, which describes when an API has been added.
+You can specify an additional `@since` tag for every parameter that has been added afterwards.
```

Comment:
"property" would be more suited than "parameter" IMHO as this applies to path & query parameters and json properties, which are all represented as TS structure properties